go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/3scale/3scale-operator
 go 1.19
 
 require (
-	github.com/3scale/3scale-porta-go-client v0.10.0
+	github.com/3scale/3scale-porta-go-client v0.9.1-0.20240208161959-a6c3b3f36473
```

Comment:
@austincunningham Do we need to downgrade here?

Reply:
hmm yea I was working of a commit sha for porta so if v0.10.0 has what I need then no not needed.  Suspect I need a release of 3scale-porta-client.

Reply:
yep need a release 
![image](https://github.com/3scale/3scale-operator/assets/16667688/9a324e94-2b00-48e2-ab14-729e160f9832)